packages/neuron-wallet/src/models/asset-account-info.ts

```diff
@@ -0,0 +1,90 @@
+import CellDep, { DepType } from "./chain/cell-dep"
+import Script, { ScriptHashType } from "./chain/script"
+import OutPoint from "./chain/out-point"
+import NetworksService from "services/networks"
+
+interface ScriptCellInfo {
+  cellDep: CellDep
+  codeHash: string
+  hashType: ScriptHashType
+}
+
+export default class AssetAccountInfo {
+  // <genesisBlockHash, sudtScriptCellInfo>
+  private sudtInfo: Map<string, ScriptCellInfo>
+  private anyoneCanPayInfo: Map<string, ScriptCellInfo>
+
+  private currentGenesisBlockHash: string
+
+  constructor(genesisBlockHash: string = NetworksService.getInstance().getCurrent().genesisHash) {
+    this.currentGenesisBlockHash = genesisBlockHash
+
+    this.sudtInfo = new Map()
+    this.anyoneCanPayInfo = new Map()
+
+    // TODO: not set yet
+    // const mainnetGenesisBlockHash = '0x92b197aa1fba0f63633922c61c92375c9c074a93e85963554f5499fe1450d0e5'
+    // const mainnetSudtCellInfo: ScriptCellInfo = {
+    //   cellDep: new CellDep(new OutPoint('', '0'), DepType.Code),
+    //   codeHash: '',
+    //   hashType: ScriptHashType.Type
+    // }
+    // const mainnetAnyoneCanPayCellInfo: ScriptCellInfo = {
+    //   cellDep: new CellDep(new OutPoint('', '0'), DepType.DepGroup),
+    //   codeHash: '',
+    //   hashType: ScriptHashType.Type
+    // }
+
+    // this.sudtInfo.set(mainnetGenesisBlockHash, mainnetSudtCellInfo)
+    // this.anyoneCanPayInfo.set(mainnetGenesisBlockHash, mainnetAnyoneCanPayCellInfo)
+
+    const testnetGenesisBlockHash = '0x63547ecf6fc22d1325980c524b268b4a044d49cda3efbd584c0a8c8b9faaf9e1'
```

Comment:
The hard coded hash and script could be moved into `const.ts`

Reply:
I think it should follow its logic.